src/platform/packages/shared/kbn-unified-tabs/src/components/tab_preview/tab_preview.tsx

```diff
@@ -62,7 +62,7 @@ export const TabPreview: React.FC<TabPreviewProps> = ({
   tabItem,
   previewData,
   stopPreviewOnHover,
-  previewDelay = 500,
+  previewDelay = 1250, // as "long" EuiToolTip delay
```

Comment:
Nit: does EUI export this value? 🤔

Reply:
no, they don't https://github.com/elastic/eui/blob/aa115fbc28a3fc107ebabbb75f6fdae75d32c976/packages/eui/src/components/tool_tip/tool_tip.tsx#L37C3-L37C17